lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1721,6 +1721,43 @@ void AtenAddTOp::getCanonicalizationPatterns(RewritePatternSet &patterns,
   });
```

Comment:
please add .mlir tests

Reply:
Thanks for the advice. I'v made the changes in the latest commit. Please review.